filebeat/prospector/log/config.go

```diff
@@ -85,7 +85,7 @@ type config struct {
 	JSON         *reader.JSONConfig      `config:"json"`
 
 	// Hidden on purpose, used by the docker prospector:
-	DockerJSON bool `config:"docker-json"`
+	DockerJSON string `config:"docker-json"`
```

Comment:
What's the reason for this change?

Reply:
I'm using the internal parameter to enable docker parsing **and** pass the desired stream through it, possible values:

- `""` - default, docker parsing is disabled
- `all` - default for `docker` prospector, read all streams
- `stdout`
- `stderr`

Reply:
Got it, didn't realise you passed through the stream param.